torch_xla/csrc/device.cpp

```diff
@@ -27,6 +27,10 @@ std::string XlaDeviceTypeToString(XlaDeviceType hw_type) {
   XLA_ERROR() << "Invalid device type";
 }
 
+// This is set when any device is initialized, so to prevent using non-virtual
+// device and virtual device together.
+static bool lock_spmd_config = false;
```

Comment:
nit, I felt like you should call this variabloe something along the line of `spmd_config_is_locked`. `lock_spmd_config` sounds like an action not a state.

Reply:
+1 will change as i rebase with the master